src/models/mentalTestSchema.js

```diff
@@ -0,0 +1,17 @@
+const moongoose = require('moongoose');
+
+const { Schema } = moongoose;
+
+const mentalTestSchema = new Schema({
+  user_id: { type: moongoose.Schema.Types.ObjectId, ref: 'User' },
+  ques_1: { type: Number, required: true },
+  ques_2: { type: Number, required: true },
+  ques_3: { type: Number, required: true },
+  ques_4: { type: Number, required: true },
+  ques_5: { type: Number, required: true },
+  ques_6: { type: Number, required: true },
+  testResult: { type: Number, default: 0 }
```

Comment:
Add a "severity" property that will have a Number (ranges from 0 - 2) and String (three possible values: "Mild", "Moderate", "Severe").